### PR TITLE
update selenium to 2.53.1, firefox to 47.0.1, chromedriver to 2.23

### PR DESCRIPTION
**Pull request by otsvetkov**
update selenium to 2.53.1, firefox to 47.0.1, chromedriver to 2.23

### CODE REVIEW

**Comment by Davidrums** [comment]
@kayabendroth

**Comment by allankilpatrick** [comment]
Would be really nice to get this merged in 😄

**Comment by allankilpatrick** [comment]
@otsvetkov I've merged this and created docker images. WFM

File: NodeChrome/Dockerfile.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key
 #==================
 # Chrome webdriver
 #==================
-ENV CHROME_DRIVER_VERSION 2.21
+ENV CHROME_DRIVER_VERSION 2.22
```
Comment (etiennetremel) [inline NodeChrome/Dockerfile.txt]: Should this be 2.23?

Comment (otsvetkov) [inline NodeChrome/Dockerfile.txt]: Look at the second commit

**Comment by twong** [comment]
@otsvetkov, will someone be updating the builds on docker hub? The latest one is 4 months old, and doesn't yet have this merge.

**Comment by mlespiau** [comment]
@allankilpatrick @otsvetkov What's missing for having this docker image released to docker hub?

**Comment by ddavison** [comment]
this was fixed

**Pull request closed without merging**